source/crud/crud.md

```diff
@@ -0,0 +1,2513 @@
+
+If there are one or more available servers and one or more of those servers is pre-5.0 (i.e. wire version \< 13),
+drivers MUST NOT use the available read preference and MUST instead select a server using a primary read preference.
+
+Otherwise, if there are either no available servers, all available servers are 5.0+ (i.e. wire version >= 13), or the
+topology type is LoadBalanced (we can assume the backing mongos is 5.0+), drivers MUST use the available read
+preference.
+
+Drivers SHOULD augment their
+[server selection algorithm](../server-selection/server-selection.md#server-selection-algorithm) such that this logic
+can be enforced within a single server selection attempt.
+
+Drivers MUST discern the read preference used to select a server for the operation, which SHALL be used for specifying
+the [$readPreference global command argument](../message/OP_MSG.rst#global-command-arguments) and
+[passing read preference to mongos and load balancers](../server-selection/server-selection.md#passing-read-preference-to-mongos-and-load-balancers)
+(if applicable).
+
+## Test Plan
+
+See the [README](tests/README.md) for tests.
+
+## Motivation
+
+Current drivers have chosen slightly different names and semantics for the same operations and options. In addition, not
+all drivers offer all the same operations and methods. As such, it is difficult to transition from driver to driver
+making the jobs of polyglot developers, documentation authors, and support engineers more difficult.
+
+## Backwards Compatibility
+
+This spec should be mostly backwards compatible as it is very lenient. Drivers finding a backwards compatibility problem
+should attempt to work around it using an acceptable deviation. In rare cases, a driver may need to break backwards
+compatibility. This should be done in accordance with a versioning scheme indicating that a backwards compatible break
+may have occurred in conjunction with release documentation and warnings.
+
+## Reference Implementation
+
+See Test Plan
+
+## Related Terminology
+
+If a driver needs to refer to items in the following list, the below are the accepted forms of those terms and
+deviations from the Naming section are still permissible.
+
+- Read Preference: readPreference
+- Read Concern: readConcern
+- Write Concern: writeConcern
+
+## Q & A
+
+Q: Why do the names of the fields differ from those defined in the MongoDB manual?\
+Documentation and commands often
+refer to same-purposed fields with different names making it difficult to have a cohesive API. In addition, occasionally
+the name was correct at one point and its purpose has expanded to a point where the initial name doesn't accurately
+describe its current function.
+
+In addition, responses from the servers are sometimes cryptic and used for the purposes of compactness. In these cases,
+we felt the more verbose form was desirable for self-documentation purposes.
+
+Q: Where is read preference?\
```

Comment:
```suggestion
Q: Where is read preference?
```
It appears on a number of the Qs, maybe it was something to do with the indentation on the original rst?

Reply:
The trailing slash is a hard line break and used to mimic the definition list syntax we had in RST. This is used in both the Q&A and Terminology sections.